apex/transformer/tensor_parallel/cross_entropy.py

```diff
@@ -81,6 +102,10 @@ def backward(ctx, grad_output):
 
         # Retreive tensors from the forward path.
         softmax, target_mask, masked_target_1d = ctx.saved_tensors
+        label_smoothing, vocab_size = ctx.label_smoothing, ctx.vocab_size
+
+        label_smoothing = label_smoothing.item()
+        vocab_size = vocab_size.item()
```

Comment:
These two are Python built-in scalar so `.item()` won't work.

Reply:
Oh sorry I should have removed those.